app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -848,15 +842,14 @@ public void deleteWeightReading(WeightReading reading) {
         return idArray;
     }
 
-    public ArrayList<Integer> getWeightReadingAsArray() {
+    public List<Double> getWeightReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
Also renamed